torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -380,17 +411,24 @@ void RRefContext::addPendingUser(
   pendingUsers_[forkId] = rref;
 }
 
-void RRefContext::delPendingUser(const ForkId& forkId) {
-  std::lock_guard<std::mutex> lock(mutex_);
-  auto iter = pendingUsers_.find(forkId);
-  TORCH_INTERNAL_ASSERT(
-      iter != pendingUsers_.end(),
-      "Inconsistent states: attempt to delete a non-exist UserRRef.");
-  pendingUsers_.erase(iter);
+void RRefContext::confirmPendingUser(const ForkId& forkId) {
```

Comment:
Any reason for making this name change? Its name collides with another `confirmPendingUser` in this file.

Reply:
Right, this is more confusing now. Let me revert back. I was thinking the name "delPendingUser” can't reflect the fact that it also adds to the `confimedUsers_` map.